index.js

```diff
@@ -107,24 +139,32 @@ function myMap(array, callback) {
 // ***** Callbacks - Question 1 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
+
+function triple(number){
+  console.log(number * 3)
+}
 
-// console.log(myMap([1,2,3,4,5], triple)) 
-// // => [3,6,9,12,15]
-//
-// console.log(myMap([2,4,6,8], triple))   
-// // => [6,12,18,24]
-// console.log("%c----------", "color: red") 
+console.log(myMap([1,2,3,4,5], triple)) 
+// => [3,6,9,12,15]
+
+console.log(myMap([2,4,6,8], triple))   
+// => [6,12,18,24]
+console.log("%c----------", "color: red") 
 
 
 // ***** Callbacks - Question 2 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 2", "color: red")
+console.log("%cCallbacks - Question 2", "color: red")
+
+function greet(name){
+console.log (`Hi, ${name}`)
+}
```

Comment:
Be careful of the difference between `console.log` and `return` in the `greet` and `triple` functions. Try running this code, and see what the array has:

```js
console.log(myMap(["Ian", "Chase"], greet))
```

You should see two console.log messages with strings, then an array with `undefined` inside - try to fix that!